src/tests/trip/delete-comment.test.js

```diff
@@ -0,0 +1,88 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import app from '../../app';
+import { loggedInToken, userWithNoTripToken } from '../fixtures/users.fixture';
+import { deleteComment, othersComment } from '../fixtures/comments.fixture';
+import { createTrip } from '../fixtures/trip.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('/DELETE delete comment posted on trip request', () => {
+  let trip;
+  before(async () => {
+    trip = await createTrip();
+  });
+  it('Should delete a comment post on thread', (done) => {
+    chai.request(app)
+      .delete(`/api/trips/requests/${trip.id}/comments/${deleteComment.id}`)
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(200);
+        res.body.should.have.property('message').equal('Comment was deleted successfully');
+        done();
+      });
+  });
+
+  it('Should check bad request passed into the parameter', (done) => {
+    chai.request(app)
+      .delete(`/api/trips/requests/${faker.lorem.word()}/comments/${deleteComment.id}`)
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(400);
+        res.body.should.have.property('message');
+        done();
+      });
+  });
+
+  it('Should not allow to delete comment of the trip that does not exists', (done) => {
+    chai.request(app)
+      .delete(`/api/trips/requests/${faker.random.number({ min: 200, max: 200 })}/comments/${deleteComment.id}`)
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(404);
+        res.body.should.have.property('message').equal(`Trip with ID ${faker.random.number({ min: 200, max: 200 })} does not exists`);
+        done();
+      });
+  });
+
+  it('Should not allow to delete comment that does not exists', (done) => {
+    chai.request(app)
+      .delete(`/api/trips/requests/${trip.id}/comments/${faker.random.number({ min: 200, max: 200 })}`)
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(404);
+        res.body.should.have.property('message').equal(`Comment with ID ${faker.random.number({ min: 200, max: 200 })} does not exists`);
```

Comment:
Faker should be in fixtures file and imported here. Do it in every single place where it is still happening